tests/perf/test_action.py

```diff
@@ -5,13 +5,26 @@
 
 from __future__ import annotations
 
+import logging
 from pathlib import Path
 
 import pytest
 
 from .benchmark import Benchmark
 from .conftest import PerfTestBase
 
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session")
+def fxt_deterministic(request: pytest.FixtureRequest) -> bool:
+    """Override the deterministic setting for action classification task."""
+    deterministic = request.config.getoption("--deterministic")
+    deterministic = True if deterministic is None else deterministic.lower() == "true"
+    msg = f"deterministic={deterministic}"
+    log.info(msg)
```

Comment:
```suggestion
    log.info(f"{deterministic=}")
```
f"{deterministic=}" is same as "deterministic=${deterministic}"

Reply:
ok. can I change it later? I'll keep in mind this compact code.